src/GovUk.Education.ExploreEducationStatistics.Content.Requests/DataSetsListRequestSortBy.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace GovUk.Education.ExploreEducationStatistics.Content.Requests;
 
-public enum DataSetsListRequestOrderBy
+public enum DataSetsListRequestSortBy
```

Comment:
Do you think we should also change the terminology in the XML documentation below, changing 'Order by' to 'Sort by'?